tests/app/test_event_handlers.py

```diff
@@ -129,3 +130,15 @@ def test_set_user_permissions(client_request, mock_events):
 
     create_set_user_permissions_event(**kwargs)
     mock_events.assert_called_with("set_user_permissions", event_dict(**kwargs))
+
+
+def test_create_url():
+    url_for_redirect = create_url(
+        "production", "https://notify.gov/using-notify/get-started"
+    )
+    assert url_for_redirect == "https://beta.notify.gov/using-notify/get-started"
+
+
+def test_create_url_non_production():
+    url = create_url("development", "https://notify.gov/using-notify/get-started")
+    assert url == "https://notify.gov/using-notify/get-started"
```

Comment:
Based on the suggestions above, we'll have to tweak these slightly:

```suggestion
def test_create_beta_url():
    url_for_redirect = create_url("https://notify.gov/using-notify/get-started"
    )
    assert url_for_redirect == "https://beta.notify.gov/using-notify/get-started"


def test_create_beta_url_for_beta_site():
    url = create_url("development", "https://beta.notify.gov/using-notify/get-started")
    assert url == "https://beta.notify.gov/using-notify/get-started"
```

We could also add one more test to check that a redirect doesn't happen if someone goes to `beta.notify.gov` directly based on @stvnrlly's suggestion:

```python
@patch.dict("current_app.config", {"NOTIFY_ENVIRONMENT": "production"})
def test_production_beta_redirect():
    ...  # call the redirect function and check that it redirects properly.


@patch.dict("current_app.config", {"NOTIFY_ENVIRONMENT": "development"})
def test_non_production_beta_redirect():
    ...  # call the redirect function and check that it does not redirect.
```